### DISCUSSION TITLE
Lesson 7 Foundry Fund Me

### DISCUSSION THREAD

**Discussion by chrisblock8** [Cyfrin/foundry-full-course-cu#994] [Q&A]
Hello everyone!!!!
I'm having an issue with the .evn file it says that there is no file with that name even thou I already have it created it, what can do?

and another problem I keep having its this one with the --fork-url....
/[
](error: a value is required for '--fork-url <URL>' but none was supplied

**Comment by hparihar-07** [comment]
@chrisblock8 Check there should be an .env file and not a .evn file

**Comment by chrisblock8** [comment]
Hey, thank you, i already made sure that  is (env) but the problem still persists  On Wed, Oct 11, 2023 at 8:04 PM, hparihar-07 ***@***.***> wrote:  
@chrisblock8 Check there should be an .env file and not a .evn file

**Reply by 0xusmanf** [reply]
Make sure you performe `source .env` command after making any changes to your `.env` file

**Comment by chrisblock8** [comment]
I have done that as well, thank you  On Thu, Oct 12, 2023 at 12:20 AM, Usman Farooq ***@***.***> wrote:  
Make sure you performe source .env command after making any changes to your .env file

**Reply by 0xusmanf** [reply]
Also, make sure you have saved `.env` file in the root directory of the project and not in some sub-directory.

**Comment by chrisblock8** [comment]
Ok that might be the problem Usman, how can I get that done?  On Thu, Oct 12, 2023 at 12:39 AM, Usman Farooq ***@***.***> wrote:  
Also, make sure you have saved .env file in the root directory and not in some sub-directory.

**Reply by 0xusmanf** [reply]
It should be here, outside of all the other folders.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/2a07873f-0ced-482c-a3ce-29f3d30be893)

**Comment by chrisblock8** [comment]
Ohh ok kool, can you explain me how can I do that   On Thu, Oct 12, 2023 at 12:49 AM, Usman Farooq ***@***.***> wrote:  
It should be here, outside of all the other folders.

**Reply by 0xusmanf** [reply]
Press and hold your `.env` file and move it to the end. Or another way is to open your project in File Explorer and cut past your `.env` in the root directory or just create a new `.env` file there. Like in the below example. It is not saved in the `test`, `src` or any other folder but in the main `fund_me_with_foundry` folder.

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110072406/383d6d47-9e78-4c30-8347-7baa1eb3e776)

**Comment by chrisblock8** [comment]
Hey, thank you I’m gonna try doing that instead.  On Thu, Oct 12, 2023 at 1:00 AM, Usman Farooq ***@***.***> wrote:  
Press and hold your .env file and move it to the end. Or another way is to open your project in File Explorer and cut past your .env in the root directory or just create a new .env file there. Like in the below example. It is not saved in the test, src or any other folder but in the main fund_me_with_foundry folder.
